test/functional/cmdLineTests/modularityddrtests/modularityddrtests.xml

```diff
@@ -630,4 +631,304 @@
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!dumpmodule requires $javaCompilerModuleAddr$</input>
+			<input>quit</input>
+		</command>
+		<saveoutput regex="no" type="required" saveName="requiredModuleAddr" splitIndex="1" splitBy="!j9module ">!j9module 0x</saveoutput>
+		<output regex="no" type="success">!j9module 0x</output>
+		<output regex="no" type="required">java.base</output>
+		<output regex="no" type="failure">DDRInteractiveCommandException</output>
+	</test>
+
+	<test id="Verify !dumpmodule requires">
+		<exec command="sh" capture="LOGNAME" platforms="zos.*" >
+			<arg>-c</arg>
+			<arg>echo $$LOGNAME</arg>
+		</exec>
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP.X001' $DUMPFILE$" platforms="zos_390-64.*" />
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!j9module $requiredModuleAddr$</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success">Fields for J9Module:</output>
+		<output regex="no" type="required">moduleName</output>
+		<output regex="no" type="required">version</output>
+		<output regex="no" type="required">readAccessHashTable</output>
+		<output regex="no" type="failure">DDRInteractiveCommandException</output>
+		<output regex="no" type="failure">&lt;FAULT&gt;</output>
+	</test>
+
+	<test id="Run !dumpmodule exports">
+		<exec command="sh" capture="LOGNAME" platforms="zos.*" >
+			<arg>-c</arg>
+			<arg>echo $$LOGNAME</arg>
+		</exec>
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP.X001' $DUMPFILE$" platforms="zos_390-64.*" />
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!dumpmodule exports $javaBaseModuleAddr$</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success">Exports java/nio/file/spi</output>
+		<output regex="no" type="required">!j9package 0x</output>
+		<output regex="no" type="required">Exports sun/util/logging</output>
+		<output regex="no" type="required">to java.logging</output>
+		<output regex="no" type="required">!j9module 0x</output>
+		<output regex="no" type="required">to ALL-UNNAMED</output>
+		<saveoutput regex="no" type="required" saveName="sunAnnotationPackageAddr" splitIndex="1" splitBy="!j9package ">sun/reflect/annotation </saveoutput>
```

Comment:
References to internal packages like `sun/util/logging` and `sun/reflect/annotation` will make this test fragile: it will break if/when they are renamed or refactored. Also, it appears `sunAnnotationPackageAddr` is unused.

Reply:
References to sun packages have been replaced with `com/ibm/gpu/spi`. `sunAnnotationPackageAddr` has been removed